docs/reference/glossary.asciidoc

```diff
@@ -24,28 +24,46 @@ Also see <<glossary-text,text>> and <<glossary-term,term>>.
 // end::analysis-def[]
 --
 
+[[glossary-auto-follow-pattern]] auto-follow pattern ::
+// tag::auto-follow-pattern-def[]
+An index pattern pattern used to automatically configure new indices as 
+<<glossary-follower-index, follower indices>> for <<glossary-ccr,{ccr}>>.
```

Comment:
I tend to remove spaces from internal links. Otherwise, the links could accidentally break when rewrapped for line length.
```suggestion
<<glossary-follower-index,follower indices>> for <<glossary-ccr,{ccr}>>.
```